tests/cypress/support/index.js

```diff
@@ -6,7 +6,7 @@ require('./commands');
 require('@cypress/code-coverage/support');
 
 before(() => {
-    if (Cypress.browser.name === 'firefox') {
+    if (Cypress.browser.family !== 'chromium') {
```

Comment:
Is this your change?

Reply:
I've merged PR from @dvkruchinin with some UI test fixes.